ci_framework/roles/rhol_crc/molecule/add_crc_creds/prepare.yml

```
@@ -0,0 +1,30 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Prepare
+  hosts: all
+  vars:
+    cifmw_basedir: "/tmp/ci-framework"
+  roles:
+    - role: test_deps
+    - role: ci_setup
+  tasks:
+    - name: Create crc kubeconfig file
+      ansible.builtin.copy:
+        dest: "{{ cifmw_basedir }}//artifacts/kubeconfig"
```

didn't see that one, sorry ! The cifmw_basedir is a top-level we may set - but it doesn't exist as-is.
```suggestion
        dest: "/tmp/ci-framework/artifacts/kubeconfig"
```